ibis/omniscidb/client.py

```diff
@@ -498,6 +506,21 @@ def _build_ast(self, expr, context):
         result = build_ast(expr, context)
         return result
 
+    def _check_execution_type(self, execution_type: int):
```

Comment:
Can we make an `EXECUTION_TYPE` enum instead of these hard-coded ints? I'm not sure how exposed these ints are already, so I'm not sure how much of a breaking change that would be. This way, the type checking would be easier.

Reply:
sounds good .. I can try that. thanks!

Reply:
@toryhaavik how about to use string instead of enum? it seems much easier and we can add a check for that and raise an error if the options available when the parameter is wrong.

the same approach used for pandas for some parameters